proposals/restructure-KF-docs-proposal.md

```diff
@@ -0,0 +1,232 @@
+This is an initial proposal, to be iterated on, following the discussion in [#2293](https://github.com/kubeflow/website/issues/2293).
```

Comment:
What do we do to guide users to use all components cohesively? Could there be a overall playbook to lead users stepping through their journey for ML end-to-end? How can users use components in the correct orders? How can they take results from one component and feed them into the next component?

Reply:
Hi @davidyuyuan, This is good feedback! I think that would be very nice to have. Could you please create an issue on Kubeflow/website as it is slightly orthogonal to this effort?

Reply:
I agree. It might be a bit more than documentation. We have many excellent components. It requires some coordination and cooperation to make components working together smoothly, expecially for users without extensive experiences.